Sources/Stevenson/SlackService.swift

```diff
@@ -54,23 +61,21 @@ public struct SlackResponse: Content {
 }
 
 public struct SlackService {
-    let requireChannel: String?
+    let token: String
 
-    public init(
-        requireChannel: String?
-    ) {
-        self.requireChannel = requireChannel
+    public init(token: String) {
+        self.token = token
     }
 
     public func handle(command: SlackCommand, on request: Request) throws -> Future<Response> {
         let metadata: SlackCommandMetadata = try attempt {
             try request.content.syncDecode(SlackCommandMetadata.self)
         }
 
-        if metadata.token != command.token {
+        guard metadata.token == token else {
             throw Error.invalidToken
         }
-        if let requireChannel = requireChannel, metadata.channelName != requireChannel {
+        guard command.allowedChannels.isEmpty || command.allowedChannels.contains(metadata.channelName) else {
             throw Error.invalidChannel(metadata.channelName)
```

Comment:
I think it would be a more useful error message to tell which channels were allowed for the command, rather than the channel (which the user calling the command already know the name of)

e.g. if you trigger `/fastlane` from `#general` it's quite useless to repeat in the error that the channel you're already in is named "general", what's of interest for the user is where they should trigger the command instead to fix their mistake.